src/crates.rs

```diff
@@ -0,0 +1,70 @@
+use crate::{
+    api,
+    commands::{Args, Result},
+};
+
+use reqwest::header;
+use serde::Deserialize;
+
+const USER_AGENT: &str = "rust-lang/discord-mods-bot";
+
+#[derive(Debug, Deserialize)]
+struct Crates {
+    crates: Vec<Krate>,
+}
+#[derive(Debug, Deserialize)]
+struct Krate {
```

Comment:
Is this a typo or is there another reason for the `K`?

Reply:
... because krate was used for the `crate` keyword so often I just always assumed you couldn't use `Crate` either :v

Reply:
I prefer to use `Crate`